docs/fastdds/use_cases/realtime/allocations.rst

```diff
@@ -146,21 +146,23 @@ By default, a full dynamic behavior is used.
 Parameters on the DataWriter
```

Comment:
I think this [section](https://fast-dds.docs.eprosima.com/en/latest/fastdds/use_cases/realtime/allocations.html#parameters-on-the-participant) should also include the information about `content_filter` allocations in the [Limiting the size of custom parameters](https://fast-dds.docs.eprosima.com/en/latest/fastdds/use_cases/realtime/allocations.html#limiting-the-size-of-custom-parameters)

Reply:
Commit 1a58046